invokeai/frontend/web/src/features/nodes/components/flow/Flow.tsx

```diff
@@ -149,10 +151,26 @@ export const Flow = () => {
     dispatch(contextMenusClosed());
   }, [dispatch]);
 
-  const onInit: OnInit = useCallback((flow) => {
-    $flow.set(flow);
-    flow.fitView();
-  }, []);
+  const onInit: OnInit = useCallback(
+    (flow) => {
+      $flow.set(flow);
+      flow.fitView();
+      flowWrapper.current = document.getElementById('workflow-editor');
+    },
+    [flowWrapper]
+  );
+
+  const onMouseMove = (event: MouseEvent) => {
+    const bounds = flowWrapper.current?.getBoundingClientRect();
+    if (bounds) {
+      const pos = $flow.get()?.project({
+        x: event.clientX - bounds.left,
+        y: event.clientY - bounds.top,
+      });
+
+      dispatch(mouseMoved(pos));
+    }
+  };
 
```

Comment:
You can import `MouseEvent` from `react`, then provide this generic with a type argument to narrow it. In this case, `HTMLDivElement`.

This saves you from these minor acrobatics: `<ReactFlow onMouseMove={(event) => event.nativeEvent} />`, satisfying the TS gods.